src/modules/chips/components/chipsContainer/ChipsContainer.js

```diff
@@ -177,11 +185,13 @@ export class ChipsContainer extends MvuElement {
 			return currentChips.map((chip) => (chip.target === 'modal' ? getButton(chip) : getLink(chip)));
 		};
 
+		const getOverlayNavClass = () => (isOpenNavigationRail && !isPortrait ? 'is-open-navigationRail' : '');
+
 		return html`
 			<style>
 				${css}
 			</style>
-			<div id="chipscontainer" class="${getOrientationClass()} ${getMinWidthClass()} ${getOverlayClass()} chips__container">
+			<div id="chipscontainer" class="${getOrientationClass()} ${getMinWidthClass()} ${getOverlayClass()} ${getOverlayNavClass()} chips__container">
```

Comment:
I suggest to collect all classes in a classMap, to have the controlflow in one place like:
```suggestion
    const classes = {
            'is-open-navigationRail':isOpenNavigationRail && !isPortrait,
            'is-desktop': hasMinWidth,
            'is-tablet':!hasMinWidth,
            'is-portrait' : isPortrait,
            'is-landscape':!isPortrait
```
}`